src/reducer.js

```diff
@@ -0,0 +1,25 @@
+const initialState = {
+  selectedRegionId: 0,
+  regions: [],
+};
+
+const reducers = {
+  setRegions: (state, payload) => ({
+    ...state,
+    regions: payload.regions,
+  }),
```

Comment:
```suggestion
  setRegions: (state, { regions }) => ({
    ...state,
    regions,
  }),
```
이런식으로 구조 분해 할당을 사용해줘도 좋을 것 같아요~

Reply:
오.. 요게 더 깔끔하네요 ㅎㅎ..